.github/workflows/build_and_test.yml

```diff
@@ -479,10 +479,9 @@ jobs:
           toolchain: stable
       - uses: actions/checkout@v3
       - uses: Swatinem/rust-cache@v2
-      - name: Run clippy
-        uses: actions-rs/cargo@v1
-        with:
-          command: clippy
+      - name: Run real clippy, not the fake one
```

Comment:
What happened here? :D

Reply:
we want to run `cargo clippy --all-features ... ` etc not `cargo clippy`